README.md

```diff
@@ -41,8 +41,15 @@ jobs:
         destChannel: "github-notifications"
 ```
 
-### Ziti Identity
+### Inputs
+
+#### `zitiId`
 
 The `zitiId` input is the JSON formatted string of an identity enrolled  in an OpenZiti Network.
 
 The identity can be created by enrolling via the `ziti edge enroll path/to/jwt [flags]` command.  The `ziti` CLI executable can be obtained [here](https://github.com/openziti/ziti/releases/latest).
+
+#### `webhookUrl`
+
+This input value is a Mattermost internal webhook that allows you to post to any channel and set the sender's name.
```

Comment:
Sugges something liket: "This input value is a Mattermost "Incoming Webhook" URL available over an OpenZiti network to the identity specified by`zitiId`.  This URL should be configured in Mattermost to allow posting to any valid channel with any sender Username (default will be the `sender.login` specified in the GitHub Action event)